stetho-repl-js/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_NAME=Stetho REPL JavaScript module
+POM_ARTIFACT_ID=stetho-repl-js
```

Comment:
I'm ok with renaming the project/package. I might be tempted at renaming it stetho-js-rhino  (we can ditch repl). This way it is clear that it is a Javascript implementation and leaves the path to other implementations.